### PR TITLE
GCP Monitoring Data Connector

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes

  - add GCP Monitoring Data Connector

### CODE REVIEW

File: Sample Data/Custom/GCP_MONITORING_CL.json [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        "metric_type_s": "compute.googleapis.com/instance/cpu/utilization",
+        "resource_type_s": "gce_instance",
+        "resource_labels_project_id_s": "sometestproject-111111",
+        "resource_labels_instance_id_s": "1111111111111111111",
+        "resource_labels_zone_s": "europe-west3-c",
+        "metricKind_s": "GAUGE",
+        "valueType_s": "DOUBLE",
+        "interval_startTime_t": "6/16/2021, 2:27:00.000 PM",
+        "interval_endTime_t": "6/16/2021, 2:27:00.000 PM",
+        "value_int64Value_d": "",
```
Comment (v-maudan) [inline Sample Data/Custom/GCP_MONITORING_CL.json:25]: please update "value_int64Value_d"  field value to double type as it is ingesting string in LAW. e.g 10, 47934

**Review by v-maudan — CHANGES_REQUESTED** [review]

File: Solutions/GoogleCloudPlatformMonitor/Data Connectors/azuredeploy_GCP_MONITOR_API_FunctionApp.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "GCPMONITOR",
```
[the remark this answers is not in the document]
Comment (vu-socprime) [reply Solutions/GoogleCloudPlatformMonitor/Data Connectors/azuredeploy_GCP_MONITOR_API_FunctionApp.json:6]: updated

File: Solutions/GoogleCloudPlatformMonitor/Data Connectors/azuredeploy_GCP_MONITOR_API_FunctionApp.json [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+                        "WORKSPACE_ID": "[parameters('AzureSentinelWorkspaceId')]",
+                        "SHARED_KEY": "[parameters('AzureSentinelSharedKey')]",
+                        "logAnalyticsUri": "[variables('LogAnaltyicsUri')]",
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://aka.ms/sentinel-GCPMonitorDataConnector-functionapp"
+                    }
+                }
+            ]
+        },
+        {
+            "type": "Microsoft.Web/sites/hostNameBindings",
```
[the remark this answers is not in the document]
Comment (vu-socprime) [reply Solutions/GoogleCloudPlatformMonitor/Data Connectors/azuredeploy_GCP_MONITOR_API_FunctionApp.json]: Updated. Thanks

File: Sample Data/Custom/GCP_MONITORING_CL.json [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+        "metric_labels_device_name_s": "",
+        "metric_labels_storage_type_s": "",
+        "metric_type_s": "compute.googleapis.com/instance/cpu/utilization",
+        "resource_type_s": "gce_instance",
+        "resource_labels_project_id_s": "sometestproject-111111",
+        "resource_labels_instance_id_s": "1111111111111111111",
+        "resource_labels_zone_s": "europe-west3-c",
+        "metricKind_s": "GAUGE",
+        "valueType_s": "DOUBLE",
+        "interval_startTime_t": "6/16/2021, 2:27:00.000 PM",
```
[the remark this answers is not in the document]
Comment (vu-socprime) [reply Sample Data/Custom/GCP_MONITORING_CL.json:23]: updated

File: Sample Data/Custom/GCP_MONITORING_CL.json (hunk 1 above)
Comment (vu-socprime) [reply Sample Data/Custom/GCP_MONITORING_CL.json]: updated

**Review by v-maudan — APPROVED** [review]

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**